File: frontend/client/lib/models/qrcode.dart
class QRCode {
  final int qrCodeId;
  final String codeValue;
  final int? poiId;

  QRCode({
    required this.qrCodeId,
    required this.codeValue,
    this.poiId,
  });

  factory QRCode.fromJson(Map<String, dynamic> json) {
    return QRCode(
      qrCodeId: json['qrCodeId'],
      codeValue: json['codeValue'],
      poiId: json['poiId'],
    );
  }
}

File: frontend_fix/client/lib/model/QRCode.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace client.lib.model
{
    public class QRCode
    {
        [JsonPropertyName("qrCodeId")]
        public int QrCodeId { get; set; }

        [JsonPropertyName("codeValue")]
        public string CodeValue { get; set; } = string.Empty;

        [JsonPropertyName("poiId")]
        public int? PoiId { get; set; }
    }
}
